tests/unit/commits/test_commits.py

```diff
@@ -0,0 +1,40 @@
+import mock
+import pytest
+from bitbucket.commits import BitbucketCommits
+from bitbucket.exceptions import BitbucketBaseAPIException
+
+
+class TestBitbucketRepos:
+    @mock.patch('bitbucket.service.BitbucketService.get')
+    def test_get_commits(self, mock_get_request):
+        repos_service = BitbucketCommits(
+            credentials_provider=mock.Mock(), project_key='the_wasp', repository_name='winsome',
+            since='7c3d7a55b847f234249936c229bd974626b08a99', merges='ONLY'
```

Comment:
Should you not use the ONLY constant here?

Reply:
Sure!